array-loops.js

```diff
@@ -32,4 +43,9 @@ const figures = [4, 6, 12, 34, 1];
 // Wanneer je de volledige array logt zie je alle nieuwe waardes
 // console.log(figures) geeft: [ 7, 9, 15, 37, 4 ]
 
+// answer: for-i loop is runs for length of "figures" and add +3 to each number from figures (figures[i])* 3 , console.log outside loop;
+for (let i = 0; i < figures.length; i++) {
+    figures[i] = figures[i] + 3;
+}
+console.log(figures);
 
```

Comment:
Helemaal goed. Een lichte verbeterpunt op je hele code zou kunnen zijn als je deze nog in functies uitschrijft zoals we in de college behandeld hadden.